testutil/src/test/java/org/spine3/test/VerifyShould.java

```diff
@@ -0,0 +1,1312 @@
+        Verify.assertStartsWith(new Integer[1]);
+    }
+
+    @Test
+    public void pass_if_array_starts_with_items() {
+        final Integer[] array = {1, 2};
+        final Integer firstItem = array[0];
+
+        Verify.assertStartsWith(array, firstItem);
+    }
+
+    @Test(expected = AssertionError.class)
+    public void fail_if_list_not_starts_with_items() {
+        final List<Integer> list = Arrays.asList(1, 2, 3);
+        final Integer notStartsWith = 777;
+
+        Verify.assertStartsWith(list, notStartsWith);
+    }
+
+    @SuppressWarnings("ConstantConditions")
+    @Test(expected = AssertionError.class)
+    public void fail_if_list_is_null_in_starts_with() {
+        Verify.assertStartsWith((List) null, 1, 2, 3);
+    }
+
+    @Test(expected = AssertionError.class)
+    public void fail_if_items_is_empty_in_list_starts_with() {
+        Verify.assertStartsWith(Collections.emptyList());
+    }
+
+    @Test
+    public void pass_if_list_starts_with_items() {
+        final List<Integer> list = Arrays.asList(1, 2, 3);
+        final Integer firstItem = list.get(0);
+
+        Verify.assertStartsWith(list, firstItem);
+    }
+
+    @Test(expected = AssertionError.class)
+    public void fail_if_list_not_ends_with_items() {
+        final List<Integer> list = Arrays.asList(1, 2, 3);
+        final Integer notEndsWith = 777;
+
+        Verify.assertEndsWith(list, notEndsWith);
+    }
+
+    @SuppressWarnings("ConstantConditions")
+    @Test(expected = AssertionError.class)
+    public void fail_if_list_is_null_in_ends_with() {
+        Verify.assertEndsWith((List) null, 1, 2, 3);
+    }
+
+    @Test(expected = AssertionError.class)
+    public void fail_if_items_is_empty_in_list_ends_with() {
+        Verify.assertEndsWith(Collections.emptyList());
+    }
+
+    @Test
+    public void pass_if_list_ends_with_items() {
+        final List<Integer> list = Arrays.asList(1, 2, 3);
+        final Integer lastItem = list.get(list.size() - 1);
+
+        Verify.assertEndsWith(list, lastItem);
+    }
+
+    @Test(expected = AssertionError.class)
+    public void fail_if_array_not_ends_with_items() {
+        final Integer[] array = {1, 2, 3};
+        final Integer notEndsWith = 777;
+
+        Verify.assertEndsWith(array, notEndsWith);
+    }
+
+    @SuppressWarnings("ConstantConditions")
+    @Test(expected = AssertionError.class)
+    public void fail_if_array_is_null_in_ends_with() {
+        Verify.assertEndsWith((Integer[]) null, 1, 2, 3);
+    }
+
+    @Test(expected = AssertionError.class)
+    public void fail_if_items_is_empty_in_array_ends_with() {
+        Verify.assertEndsWith(new Integer[1]);
+    }
+
+    @Test
+    public void pass_if_array_ends_with() {
+        final Integer[] array = {1, 2, 3};
+        final Integer lastItem = array[array.length - 1];
+
+        Verify.assertEndsWith(array, lastItem);
+    }
+
+    @Test(expected = AssertionError.class)
+    public void fail_if_objects_are_not_equal() {
```

Comment:
You should describe what operation should fail in the method name. It's not clear now. The only thing which is clear is that _something_ should fail if objects aren't equal. 

It's here and in similar cases below.